s/frontend/components/modals/views/delete-thread/delete-thread-modal-view.ts

```diff
@@ -0,0 +1,37 @@
+
+import {html} from "lit"
+import {ModalControls} from "../../modal-types.js"
+import {NestedComment} from "../../../../models/commenting/commenting-types.js"
+
+export async function deleteThreadModalView({
+		modals,
+		comment,
+	}: {
+		modals: ModalControls
+		comment: NestedComment
+	}) {
+
+		const isThread = comment.parentCommentId === undefined
+		const isReview = comment.scoring !== undefined
+
+		const deleteText = isThread
+			? isReview
+				? "review"
+				: "thread"
+			: "reply"
+
+		const result = await modals.confirm({
+			closeOnBlanketClick: true,
+			renderYes: () => html`Delete`,
+			renderNo: () => html`Cancel`,
+			renderContent: () => html`
+			<div class="modalview report">
```

Comment:
see the class "report" here? that should only apply to "report user" modal view.

this should have a class like `modalview deletepost` or something like that